manifests/v/VMware/WorkstationPlayer/17.5.0/VMware.WorkstationPlayer.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: VMware.WorkstationPlayer
+PackageVersion: 17.5.0
+PackageLocale: en-US
+Publisher: VMware, Inc.
+PublisherUrl: https://www.vmware.com/products/workstation-player.html
+PublisherSupportUrl: https://www.vmware.com/support/player.html
+PrivacyUrl: https://www.vmware.com/help/privacy.html
+Author: VMware, Inc.
+PackageName: VMware Workstation Player
+PackageUrl: https://www.vmware.com/products/workstation-player.html
+License: Proprietary
+LicenseUrl: https://www.vmware.com/help/legal.html
+Copyright: Copyright © 2023 VMware, Inc. All rights reserved.
+CopyrightUrl: https://www.vmware.com/help/trademarks.html
+ShortDescription: VMware Workstation Player™ is a stable, mature solution for local desktop virtualization, allowing you to safely run a second operating system as a virtual machine on a single PC.
+Description: VMware Workstation Player™ is a streamlined desktop virtualization application that runs another operating system on the same computer without rebooting. VMware Workstation Player provides a simple user interface, unmatched operating system support, and portability across the VMware ecosystem.
+Moniker: vmware-workstation-player
+Tags:
+- hypervisor
+- virtualmachine
+- virtual-machine
+# Agreements:

```

```suggestion
Agreements:
  - AgreementLabel: VMWARE END USER LICENSE AGREEMENT
    AgreementUrl: https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/downloads/eula/universal_eula.pdf
```

Requires agreements as EULA switch is used for a silent install